services/api-server/src/simcore_service_api_server/services/webserver.py

```diff
@@ -284,16 +295,15 @@ async def clone_project(self, project_id: UUID) -> ProjectGet:
         return ProjectGet.parse_obj(result)
 
     async def get_project(self, project_id: UUID) -> ProjectGet:
-        response = await self.client.get(
-            f"/projects/{project_id}",
-            cookies=self.session_cookies,
-        )
-
-        data = self._get_data_or_raise(
-            response,
-            {status.HTTP_404_NOT_FOUND: ProjectNotFoundError(project_id=project_id)},
-        )
-        return ProjectGet.parse_obj(data)
+        with _handle_webserver_api_errors():
```

Comment:
thx. this is much better than the old _get_data_or_raise. We should remove that at some point.